### ISSUE TITLE
RNNLM seems to be working in training mode when recognizing with PyTorch

### ISSUE THREAD

**Issue by takaaki-hori** [espnet/espnet#230]
When using `asr_recog.py --backend pytorch`, the RNNLM module seems to be used in training mode, which means that the dropout function is still active even in the recognition phase. 
I think we need to put `rnnlm.eval()` somewhere. Can someone fix this and check the performance difference?
